src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -160,20 +206,20 @@ SyntaxNodeAnalysisContext ctx
 			return context;
 		}
 
-		private static bool IsExpressionTree( SyntaxNode node, SemanticModel model ) {
-			// Expression trees aren't compatible with named arguments,
-			// so skip any expressions
-			// Only lambda type expressions have arguments,
-			// so this only applies to LambdaExpression
+		private static bool IsExpressionTree( SyntaxNode node, SemanticModel model ) {
+			// Expression trees aren't compatible with named arguments,
+			// so skip any expressions
+			// Only lambda type expressions have arguments,
+			// so this only applies to LambdaExpression
 			var expressionType = model.Compilation.
 				GetTypeByMetadataName( "System.Linq.Expressions.LambdaExpression" );
 
 			if( expressionType == null || expressionType.Kind == SymbolKind.ErrorType ) {
 				return false;
-			}
-
-			// the current call could be nested inside an expression tree, so
-			// check every call we are nested inside
+			}
+
+			// the current call could be nested inside an expression tree, so
+			// check every call we are nested inside

```

Comment:
In the future it would be preferable to do whitespace changes in their own PR :)

Reply:
I don't know what is happening here... I tried to get rid of these but they don't show up on my local.